fenix/app/src/main/java/org/mozilla/fenix/search/SearchFragmentStore.kt

```diff
@@ -100,6 +100,10 @@ sealed class SearchEngineSource {
  * @property showSessionSuggestionsForCurrentEngine Whether or not to show local tabs suggestions for only
  * the current search engine.
  * @property showAllSessionSuggestions Whether or not to show the session suggestion in the AwesomeBar.
+ * @property showSponsoredSuggestions Whether or not to show sponsored Firefox Suggest search suggestions in the
+ * AwesomeBar.
+ * @property showNonSponsoredSuggestions Whether or not to show Firefox Suggest search suggestions for web content
+ * in the AwesomeBar.
```

Comment:
nit: Do we have captured anywhere in comments that we're only showing suggestions for the default? I think here or the `AwesomeBarView` code could benefit from a reminder about that particular domain detail.